docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRestore.md

```diff
@@ -0,0 +1,202 @@
+---
+description: The Remove-NetworkControllerRestore cmdlet removes a Network Controller database restore object.
+external help file: Microsoft.NetworkController.Powershell.dll-help.xml
+Module Name: NetworkController
+ms.date: 09/27/2021
+online version: https://docs.microsoft.com/powershell/module/networkcontroller/remove-networkcontrollerrestore?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Remove-NetworkControllerRestore
+---
+
+# Remove-NetworkControllerRestore
+
+## SYNOPSIS
+Removes a Network Controller database restore object.
+
+## SYNTAX
+
+```
+Remove-NetworkControllerRestore [-ResourceId] <String> [[-Etag] <String>] [-Force] -ConnectionUri <Uri>
+ [-CertificateThumbprint <String>] [-Credential <PSCredential>] [-PassInnerException] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Remove-NetworkControllerRestore** cmdlet removes a Network Controller database restore object.
+
+## EXAMPLES
+
+### Example 1: Remove a Network Controller database restore
+```powershell
+Remove-NetworkControllerRestore -ResourceId $ResourceId -ConnectionUri https://networkcontroller
+```
+
+This example removes the database restore object.
+The `$ResourceId` value was created with the restore action by using the [New-NetworkControllerRestore](New-NetworkControllerRestore.md) cmdlet.
+
+## PARAMETERS
+
+### -CertificateThumbprint
+Specifies the digital public key X.509 certificate of a user account that has permission to perform this action.
+Specify this parameter only if you run this cmdlet on a computer that is not part of the network controller cluster.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConnectionUri
+Specifies the Uniform Resource Identifier (URI) of a Network Controller.
+The cmdlet gets the database restore for that controller.
```

Comment:
Probably this description belongs to the `Get-` cmdlet

Reply:
Fixing to say removes.